Builder/DatagridBuilder.php

```diff
@@ -151,4 +168,23 @@ public function getBaseDatagrid(AdminInterface $admin, array $values = array())
 
         return new Datagrid($admin->createQuery(), $admin->getList(), $pager, $formBuilder, $values);
     }
+
+    /**
+     * Get pager by pagerType
+     *
+     * @param string $pagerType
+     *
+     * @return \Sonata\AdminBundle\Datagrid\PagerInterface
+     *
+     * @throws \RuntimeException If invalid pager type is set.
+     */
+    protected function getPager($pagerType)
```

Comment:
I tried to keep the same logic of MySQL PHP file for be easier to mantain both projects by just comparing them -- till common areas such this one become merged into Admin bundle